build.override.targets

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0"  xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--if ServiceUri is set, the service has been already started on another machine-->
+  <Target Name="WcfSetup" Condition="($(WithCategories.Contains('OuterLoop')) or '$(OuterLoop)' == 'true') and ('$(ServiceUri)' == '') and $(OS.Contains('Windows'))" BeforeTargets="BuildAllProjects">
+    <Exec Command="$(MSBuildThisFileDirectory)\src\System.Private.ServiceModel\tools\scripts\WCFCustomSetup.cmd" ConsoleToMSBuild="true">
+      <Output TaskParameter="ConsoleOutput" PropertyName="WcfSetupOutput" />
+    </Exec>
+    <Message Text="WcfSetupOutput: $(WcfSetupOutput)" />
+  </Target>
+
+  <Target Name="WcfCleanUp" Condition="($(WithCategories.Contains('OuterLoop')) or '$(OuterLoop)' == 'true') and ('$(ServiceUri)' == '') and $(WcfSetupOutput.Contains('WCFServiceCleanupNeeded')) and $(OS.Contains('Windows'))" AfterTargets="Test">
```

Comment:
Can you use the ExitCode from the WcfSetup Exec call instead of checking console output?  We went through multiple iterations with the Bridge version to simplify it to use only the exit code (0 = already running, 1 = started).  I think the use of a magic string inside the console output could be fragile and not obvious if people modify that script.